_posts/2024-12-02-ranger.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "The long journey to Apache Ranger"
+author: "Manfred Moser"
+excerpt_separator: <!--more-->
+image: /assets/images/logos/apache-ranger.png
+---
+
+[Apache Ranger]({{site.baseurl}}/ecosystem/add-on.html#apache-ranger) has
+arrived! With the new [Trino
+466]({{site.baseurl}}/docs/current/release/release-466.html) you all get another
+jam-packed release of Trino awesomeness. One of the goodies is a new plugin for
+access control for your data with Apache Ranger, and it has gone through a long
+story to get here.
```

Comment:
I think it'd be good to expand on what's new and why a Trino user would care a little longer here. We're diving into the history right after, so to make it compelling, you want to describe the value up front before getting into all that storytelling.